js/src/common/Application.js

```diff
@@ -211,7 +211,8 @@ export default class Application {
   }
 
   updateTitle() {
-    document.title = (this.titleCount ? `(${this.titleCount}) ` : '') + (this.title ? this.title + ' - ' : '') + this.forum.attribute('title');
+    document.title =
+      (this.titleCount ? `(${this.titleCount}) ` : '') + (this.title && m.route() !== '/' ? this.title + ' - ' : '') + this.forum.attribute('title');
```

Comment:
Perhaps we want to make this easier to read?

Add `count`, `separator`, and `title` variables that replace these parentheses, and just add them together `count + separator + title` when setting to document.title.

Reply:
You know what's funny? I just realized we could use the class list util (or classnames lob) to simplify this code further. Obv were not gonna do it, but... that's funny.